### PR TITLE
modify comments in pkg/apiserver/authentication/identityprovider/identity_provider.go

### PR DESCRIPTION
**Pull request by nanjofan**
### What type of PR is this?

Modify the comment error

### What this PR does / why we need it:
There are two functions of the comment was wrong.

### Special notes for reviewers:
```
Hello,This is my first attempt to submit PR.There are two functions of the comment was wrong.You can close this PR, if you modify it in another PR.THANKS.
```

### release-note
```release-note
None
```

### Additional documentation, usage docs, etc.:
```docs
None
```

### CODE REVIEW

File: pkg/apiserver/authentication/identityprovider/identity_provider.go [hunk 1]
```diff
@@ -100,7 +100,7 @@ func RegisterOAuthProvider(factory OAuthProviderFactory) {
 	oauthProviderFactories[factory.Type()] = factory
 }
 
-// RegisterOAuthProvider register GenericProviderFactory with the specified type
+// RegisterGenericProvider register GenericProviderFactory with the specified type
```
Comment (yuswift) [inline pkg/apiserver/authentication/identityprovider/identity_provider.go:103]: I think `registers` is better than `register`.

**Comment by yuswift** [comment]
/retitle modify comments in pkg/apiserver/authentication/identityprovider/identity_provider.go

File: pkg/apiserver/authentication/identityprovider/identity_provider.go (hunk 1 above)
Comment (wansir) [inline pkg/apiserver/authentication/identityprovider/identity_provider.go:103]:
```suggestion
// RegisterGenericProvider registers GenericProviderFactory with the specified type
```

**Comment by wansir** [comment]
Thanks for your contribution.

/lgtm
/approve

**Pull request merged**